src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -136,6 +138,12 @@ public InternalComponents map(EomFile eomFile, String transactionId, Date lastMo
                 throw new MethodeArticleNotEligibleForPublishException(uuid);
             }
 
+            String overrideOriginalStr = xPath.evaluate(OVERRIDE_ORIGINAL_ATTR_XPATH, attributesDocument);
+            if (sourceCode.equals(SourceCode.CONTENT_PLACEHOLDER) && overrideOriginalStr.equals("false")) {
```

Comment:
Here NullPointer exception an occur if `overrideOriginalStr` is null.
Use static import of StringUtils.isNotBlank(overrideOriginalStr) in order to become like this:
```
if (sourceCode.equals(SourceCode.CONTENT_PLACEHOLDER) && isNotBlank(overrideOriginalStr) && overrideOriginalStr.equals("false")) 
```